README.md

```diff
@@ -1 +1,85 @@
-# CapstoneJS
\ No newline at end of file
+# CapstoneJS
+
+In this project we will prepare a Kanban board with a GitHub project, with my Learning partner. It's javascript capstone project with external API..
+
+![image] ![screenshot](photo.jpg)
+
+## Video link
```

Comment:
All the members should appear in the video and talk about the project.
The video should contain a highlight about a piece of the code the students find interesting or are proud of making.

Reply:
Changes made.